src/Build/Logging/FancyLogger/FancyLoggerProjectNode.cs

```diff
@@ -26,6 +27,7 @@ private static string GetUnambiguousPath(string path)
         public string ProjectPath;
         public string TargetFramework;
         public bool Finished;
+        public string ProjectOutputExecutable;
```

Comment:
IMO this should be

```suggestion
        public string? ProjectOutputExecutable;
```

and assign/check for `null` instead of `Length > 0` elsewhere.